### DISCUSSION TITLE
⛔ 🔗 Add URL link to individual Elementor Pro Multiple Gallery images (Duplicate of #1801)

### DISCUSSION THREAD

**Discussion by miwilder** [elementor/elementor#24453] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Add URL link to individual Elementor Pro Multiple Gallery images. 
I have tried all the "addons" and none of them work.

### Describe the solution you'd like

Add URL link to individual Elementor Pro Multiple Gallery images. 

### Describe alternatives you've considered

I have tried all the "addons" and none of them work.

**Comment by nicholaszein** [comment]
Hello @miwilder!

Thank you for submitting your suggestion! 🙏 

Unfortunately, this is something we are not planning to implement due to some limitations that I've explained in [this comment](https://github.com/elementor/elementor/issues/1801#issuecomment-2283196197).

Therefore I'm closing this request. I hope you can understand!

Kind regards
